spec/models/user_spec.rb

```diff
@@ -371,5 +371,41 @@
         end
       end
     end
+
+    describe '#first_and_last_name' do
```

Comment:
This change is on master already, I don't think it should be present in this PR :thinking: